keps/sig-storage/20200206-namespace-transfer.md

```diff
@@ -0,0 +1,217 @@
+
+Check these off as they are completed for the Release Team to track. These checklist items _must_ be updated for the enhancement to be released.
+
+- [ ] kubernetes/enhancements issue in release milestone, which links to KEP (this should be a link to the KEP location in kubernetes/enhancements, not the initial KEP PR)
+- [ ] KEP approvers have set the KEP status to `implementable`
+- [ ] Design details are appropriately documented
+- [ ] Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] Graduation criteria is in place
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+**Note:** Any PRs to move a KEP to `implementable` or significant changes once it is marked `implementable` should be approved by each of the KEP approvers. If any of those approvers is no longer appropriate than changes to that list should be approved by the remaining approvers and/or the owning SIG (or SIG-arch for cross cutting KEPs).
+
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://github.com/kubernetes/enhancements/issues
+[kubernetes/kubernetes]: https://github.com/kubernetes/kubernetes
+[kubernetes/website]: https://github.com/kubernetes/website
+
+## Summary
+
+This KEP proposes an API to securely transfer Kubernetes stroage resources between namespaces.
+It will focus mainly on PersistentVolumeClaims but could apply to VolumeSnapshots as well.
+
+`StorateTransferRequest` resources are created in the target namespace.  Corresponding 
+`StorageTransferApproval` resources are created in the source namespace.  When a controller
+detects matching request/approval resources, a transfer in ititiated.
+
+When transferring a PersistentVolumeClaim, the corresponding PersistentVolume will be updated, 
+the source PersistentVolumeClaim will be deleted, and the target PersistentVolumeClaim will be created 
+(not necesserarily in that order).  The transfer simply deals with Kubernetes API resources.
+No data on the physical PersistentVolume is accessed/copied. 
+
+## Motivation
+
+Give Kubernetes users the flexability to easily and securely share data between namespaces.  
+
+### Goals
+
+- Define an API for transferring persistent storage resources between namespaces
+- The API should be compatible with as many existing provisioner implementations as possible
+
+### Non-Goals
+
+- Transfer of resources other than PersistentVolumeClaims and VolumeSnapshots will not be discussed
+- Implementation details only discussed in the context of influencing the API definition
```

Comment:
Smart, I think that's where I went wrong assuming that a POC and an implementation would be helpful; turns out it's distracting to most I think.